### DISCUSSION TITLE
Icepak post processing using pyAEDT

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#4890] [Q&A] [answered]
In pyAEDT i am able to create temperature contours using the command cepak.post.create_fieldplot_surface() , the contours that are created are with default view, now i want it to customize the view across X,Y,Z, isometric view depending on the model, is there any option for the same? 

Also, I want to use zoom to fit option before exporting the contours image. Are there any options for it?

**Comment by lorenzovecchietti** [comment, marked as answer]
If you want to export the images, you have all the options using the method here> https://github.com/ansys/pyaedt/blob/a24bfdbac699943aca7ac82765d3819326aa055f/pyaedt/modules/solutions.py#L3746

o=ipk.post.create_fieldplot_surface(...) 
o.export_image(...)

You cannot just control the view in the interface from code.
